sirepo/feature_config.py

```diff
@@ -77,6 +77,16 @@ def cfg():
     return _cfg or _init()
 
 
+def dynamic_sim_type_packages():
+    """Names of packages that contain the dynamic sim types"""
+    return frozenset(cfg().dynamic_sim_types.values())
```

Comment:
This needs to be a list (tuple), not a set, because it is ordered.

Reply:
removed 58607069